js/sudoku.js

```
@@ -39,212 +39,133 @@
 				
-				//populate table with random number depending on the level difficulty 
 				var items = defaults.level;
 				while (items > 0) {
-					var row = Math.floor(Math.random() * (8 - 0 + 1)) + 0;
-					var col = Math.floor(Math.random() * (8 - 0 + 1)) + 0;
-					if (defaults.domMatrix[row][col].children().length == 0) {
-						defaults.domMatrix[row][col].append("<div class='sdk-solution'>"+ defaults.matrix[row][col] +"</div>");
-						defaults.anwerTracker[defaults.matrix[row][col].toString()]--;
-						items--;
-					}
+					var row = Math.floor(Math.random() * 9);
```

Biome (2.1.2)

[error] Shouldn't redeclare 'row'. Consider to delete it or rename it.

'row' is defined here:


(lint/suspicious/noRedeclare)